### PR TITLE
Implement Vue Starter

### PR DESCRIPTION
**Pull request by alena-ulasava**

### CODE REVIEW

**Review by jqueryisamonad — COMMENTED** [review]
Please update `socketIo.service.js` and `signalR.service.js`

https://github.com/paralect/koa-react-starter/pull/82

File: src/App.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
```
Comment (IvanTrutnev) [inline src/App.vue]: As i know in vue 3 isn't needed wrapped element
https://v3.vuejs.org/guide/migration/fragments.html#_2-x-syntax

File: src/pages/reset/reset.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          autocomplete="off"
+        />
+      </a-form-item>
+
+      <a-form-item>
+        <a-button
+          type="primary"
+          html-type="submit"
+          :disabled="isLoading"
+          @click="handleSubmit"
```
Comment (IvanTrutnev) [inline src/pages/reset/reset.vue]: maybe should add `handleSubmit` to `<a-form>` element and remove from here?

File: src/pages/sign-in/sign-in.vue [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+          autocomplete="off"
+        />
+      </a-form-item>
+
+      <a-form-item>
+        <a-button
+          type="primary"
+          html-type="submit"
+          :disabled="isLoading"
+          @click="handleSubmit"
```
Comment (IvanTrutnev) [inline src/pages/sign-in/sign-in.vue]: the same as above

File: src/pages/sign-up/sign-up.vue [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+            autocomplete="off"
+          />
+        </a-form-item>
+
+        <a-form-item>
+          <a-button
+            type="primary"
+            html-type="submit"
+            :disabled="isLoading"
+            @click="handleSubmit"
```
Comment (IvanTrutnev) [inline src/pages/sign-up/sign-up.vue]: the same as above

**Review by IvanTrutnev — CHANGES_REQUESTED** [review]

File: src/pages/reset/reset.vue (hunk 2 above)
Comment (alena-ulasava) [reply src/pages/reset/reset.vue]: I don't like submit form on Enter :)

Comment (IharKrasnik) [reply src/pages/reset/reset.vue]: @elena-vlasova 
Why? Submit on enter is common for any form and also accessibility friendly

I agree with @IvanTrutnev that all form should be submittable on Enter press 

https://stackoverflow.com/questions/57064691/is-implicit-submission-submit-on-enter-key-useful-to-people-using-readers-and

Comment (alena-ulasava) [reply src/pages/reset/reset.vue]: Fixed

**Review by IvanTrutnev — APPROVED** [review]

**Pull request merged**